app/src/components/integrations/integrationProviders/jiraIntegration/jiraConnectionFormFields/jiraConnectionFormFields.jsx

```diff
@@ -90,7 +90,7 @@ export class JiraConnectionFormFields extends Component {
         <FieldElement
           name="url"
           label={formatMessage(COMMON_BTS_MESSAGES.linkToBtsLabel)}
-          validate={commonValidators.btsUrl}
+          validate={commonValidators.btsJiraUrl}
```

Comment:
This validation should be done for Jira **Cloud**, not Jira Server as the Jira Server can be hosted on any domain.
![image](https://github.com/user-attachments/assets/daacdc70-5ded-4311-8a94-19c66aefec2a)
So, please update validation in [Jira Cloud plugin](https://github.com/reportportal/plugin-bts-jira-cloud).

Reply:
Done.